src/python/pants/backend/python/dependency_inference/BUILD

```diff
@@ -1,7 +1,7 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources()
+python_sources(dependencies=["src/python/pants/backend/python/dependency_inference/scripts"])
```

Comment:
Asset inference no longer works, since I added a convenience function to read the now-multiple files.

Reply:
Yeah one TODO I have is to find the je ne said quoi of having directories as a single target.

In fact, once we have a unified target for assets, we could just have `asset_dir`.

There's several places in our codebase we treat entire dirs as a single unit and would love the simplicity of inference. Also less Pants has to think about.